packages/perseus/src/widgets/label-image/score-label-image.test.ts

```diff
@@ -73,138 +77,224 @@ describe("scoreMarker", function () {
 
 describe("scoreLabelImage", function () {
     it("should not grade non-interacted widget", function () {
-        const state = {
+        const userInput = {
             markers: [
                 {
-                    ...emptyMarker,
+                    ...emptyUserInput,
```

Comment:
I think once we update this object to only have what's in the user input type, we might not need it. It might be more clear to inline the single `selected: []` bit here instead of spreading this object at that point. I'll leave it to you to decide which way you want to go. :)

Reply:
Updated the tests to remove these empty types.